### PR TITLE
Feat/native/send fees improved

### LINKED ISSUE #14240: Send coins - fee levels improvements
- [ ] Fee-level cards display time estimation when the transaction should be approximately mined. ATM, the estimates are static strings. Replace it with some heuristic calculation.  (Implementation should be reused from the desktop Suite)
- [ ] the fee levels reflect the updated design (every fee level is a separate card) [FIGMA LINK](https://www.figma.com/design/sqKFZAIvk3zERxiWgthK8K/Send?node-id=2807-11437&t=VyETZijz1mTRqVWR-4)
- [ ] sat/vByte unit is displayed for Bitcoin-like coins
- [ ] In case that is selected a fee that is too high to be sent, show UI response according to [this design](https://www.figma.com/design/sqKFZAIvk3zERxiWgthK8K/Send?node-id=2807-14735&t=VPzuMJ81cT2t7tyJ-4)
   - [ ] [this animation](https://www.figma.com/proto/sqKFZAIvk3zERxiWgthK8K/Send?page-id=0%3A1&node-id=2807-14663&node-type=frame&viewport=-12257%2C-481%2C0.71&t=qGIuVxLOcOml3SEh-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=2225%3A19415) is included

### PR DESCRIPTION
**Pull request by PeKne**
## Related Issue

Resolve #14240 

## Screenshots:

https://github.com/user-attachments/assets/4f43142f-f8a9-4919-9b4e-012271237003

### CODE REVIEW

File: suite-common/wallet-core/src/fees/feesReducer.ts [hunk 1]
```diff
@@ -40,3 +41,33 @@ export const feesReducer = createReducer(initialState, builder => {
...
+    if (!networkFeeInfo) return null;
+
+    const feeLevel = networkFeeInfo.levels.find(x => x.label === level);
+
+    if (!feeLevel) return null;
+
+    return formatDuration(networkFeeInfo.blockTime * feeLevel.blocks * 60);
+};
+
+export const selectNetworkFeeLevelFeePerUnit = (
```
Comment (vytick) [inline suite-common/wallet-core/src/fees/feesReducer.ts]: The code in `selectNetworkFeeLevelTimeEstimate` and `selectNetworkFeeLevelFeePerUnit` looks very similar. It looks like a candidate for separate selector, that can be reused. Wdyt?

File: suite-native/module-send/src/components/FeesFooter.tsx [hunk 2]
```diff
@@ -8,75 +14,99 @@ import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
...
 
 type FeesFooterProps = {
+    isSubmittable: boolean;
     onSubmit: () => void;
     networkSymbol: NetworkSymbol;
     totalAmount: string;
 };
 
 const CARD_BOTTOM_PADDING = 40;
+const ENTERING_DELAY = 45;
```
Comment (vytick) [inline suite-native/module-send/src/components/FeesFooter.tsx]: nit:
```suggestion
const BUTTON_ENTERING_DELAY = 45;
```

File: suite-common/wallet-core/src/fees/feesReducer.ts (hunk 1 above)
Comment (PeKne) [reply suite-common/wallet-core/src/fees/feesReducer.ts]: makes sense 👍 3a2264530a

File: suite-native/module-send/src/components/FeesFooter.tsx (hunk 2 above)
Comment (PeKne) [reply suite-native/module-send/src/components/FeesFooter.tsx]: 3a2264530a

File: suite-common/wallet-core/src/fees/feesReducer.ts [hunk 3]
```diff
@@ -40,3 +42,39 @@ export const feesReducer = createReducer(initialState, builder => {
...
+    state: FeesRootState,
+    level: FeeLevelLabel,
+    networkSymbol?: NetworkSymbol,
+): string | null => {
+    const feeLevel = selectNetworkFeeLevel(state, level, networkSymbol);
```
Comment (vytick) [inline suite-common/wallet-core/src/fees/feesReducer.ts:76]: nit  
```suggestion
 => selectNetworkFeeLevel(state, level, networkSymbol)?.feePerUnit ?? null
```

resolve if you feel like that :)

**Review by vytick — APPROVED** [review]
Looks good :) 🚀

File: suite-common/wallet-core/src/fees/feesReducer.ts (hunk 3 above)
Comment (PeKne) [reply suite-common/wallet-core/src/fees/feesReducer.ts:76]: I prefer to not directly access props of a function call. Might lead to unreadable code. So I will resolve this.

**Pull request merged**